docs/tutorials/hello-world.rst

```diff
@@ -30,6 +30,12 @@ save it as ``rockcraft.yaml``:
 .. literalinclude:: code/hello-world/rockcraft.yaml
     :language: yaml
 
+This file permit to build a ROCK with only the ``hello`` package slice inside,
+using :ref:`chisel_explanation`. For more information
+about the ``parts`` section, check :ref:`ref_parts`.
+The other fields of the file are `metadata
+<https://canonical-rockcraft.readthedocs-hosted.com/en/latest/reference/
```

Comment:
Would it be possible to use
> :doc:`/reference/rockcraft.yaml`
instead of a link to the site?

Reply:
Yeah, it's possible and done